packages/angular/README.md

```diff
@@ -1,32 +1,29 @@
-# Installation and Code Generation
+# @coveo/angular
 
-The `@coveo/angular` package contains the necessary components to set up a search page with Angular CLI.
+The `@coveo/angular` package contains the necessary components to set up a search page using [Coveo Headless](https://docs.coveo.com/headless) and [Angular Material](https://material.angular.io/) with the [Angular CLI](https://cli.angular.io/).
 
-## Install Schematic
+If you want to create a new Coveo Headless-powered search page with the Angular web framework from scratch, it is simpler to use the [`coveo ui:create:angular` command in the main Coveo CLI](https://github.com/coveo/cli/blob/master/packages/cli/README.md#coveo-uicreateangular-name). The main Coveo CLI will handle a lot of complexity for you. The `@coveo/angular` package is rather meant to help you integrate a search page to an already existing Angular project using the Angular CLI.
```

Comment:
`main Coveo CLI` sound off to me 🤔 why main?

Reply:
Good point, I think you're right.
I initially considered the `ng add @coveo/angular --org-id<org-id> --api-key=<api-key>` command to be a kinda secondary CLI, but that's arguably misleading, I'll just rephrase.